frontend/appflowy_flutter/lib/workspace/presentation/home/hotkeys.dart

```diff
@@ -2,45 +2,95 @@ import 'dart:io';
 
 import 'package:appflowy/workspace/application/appearance.dart';
 import 'package:appflowy/workspace/application/home/home_setting_bloc.dart';
+import 'package:appflowy/workspace/application/tabs/tabs_bloc.dart';
 import 'package:flutter/material.dart';
 import 'package:hotkey_manager/hotkey_manager.dart';
 import 'package:provider/provider.dart';
 
+typedef KeyDownHandler = void Function(HotKey hotKey)?;
+
+class HotKeyItem {
+  final HotKey hotKey;
+  final KeyDownHandler keyDownHandler;
+
+  HotKeyItem({
+    required this.hotKey,
+    this.keyDownHandler,
+  });
+
+  void register() =>
+      hotKeyManager.register(hotKey, keyDownHandler: keyDownHandler);
+}
+
 class HomeHotKeys extends StatelessWidget {
   final Widget child;
   const HomeHotKeys({required this.child, Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    final HotKey hotKey = HotKey(
-      KeyCode.backslash,
-      modifiers: [Platform.isMacOS ? KeyModifier.meta : KeyModifier.control],
-      // Set hotkey scope (default is HotKeyScope.system)
-      scope: HotKeyScope.inapp, // Set as inapp-wide hotkey.
-    );
-    hotKeyManager.register(
-      hotKey,
-      keyDownHandler: (hotKey) {
-        context
-            .read<HomeSettingBloc>()
-            .add(const HomeSettingEvent.collapseMenu());
-      },
-    );
-
-    final HotKey hotKeyForToggleThemeMode = HotKey(
-      KeyCode.keyL,
-      modifiers: [
-        Platform.isMacOS ? KeyModifier.meta : KeyModifier.control,
-        KeyModifier.shift,
-      ],
-      scope: HotKeyScope.inapp,
-    );
-    hotKeyManager.register(
-      hotKeyForToggleThemeMode,
-      keyDownHandler: (_) {
-        context.read<AppearanceSettingsCubit>().toggleThemeMode();
-      },
-    );
+    // Collapse sidebar menu
+    HotKeyItem(
+      hotKey: HotKey(
+        KeyCode.backslash,
+        modifiers: [Platform.isMacOS ? KeyModifier.meta : KeyModifier.control],
+        // Set hotkey scope (default is HotKeyScope.system)
+        scope: HotKeyScope.inapp, // Set as inapp-wide hotkey.
+      ),
+      keyDownHandler: (_) => context
+          .read<HomeSettingBloc>()
+          .add(const HomeSettingEvent.collapseMenu()),
+    ).register();
```

Comment:
This appears a bit confusing.

Typically, we use `manager.register(item)` instead of `item.register()`.

Reply:
The reason for this is because of how I view the usage of this in the future.

We might have more hotkeys, and we might have hotkeys related to specific features, similar to how we have it now.

Eg. I might have this array somewhere related to the `Tabs` feature:
```
final tabsHotKeys = [
  HotKeyItem(
    hotKey: HotKey(
      KeyCode.keyW,
      modifiers: [Platform.isMacOS ? KeyModifier.meta : KeyModifier.control],
      scope: HotKeyScope.inapp,
    ),
    keyDownHandler: (_) =>
        context.read<TabsBloc>().add(const TabsEvent.closeCurrentTab()),
  ),
  HotKeyItem(
    hotKey: HotKey(
      KeyCode.pageUp,
      modifiers: [Platform.isMacOS ? KeyModifier.meta : KeyModifier.control],
      scope: HotKeyScope.inapp,
    ),
    keyDownHandler: (_) => _selectTab(context, -1),
  ),
  HotKeyItem(
    hotKey: HotKey(
      KeyCode.pageDown,
      modifiers: [Platform.isMacOS ? KeyModifier.meta : KeyModifier.control],
      scope: HotKeyScope.inapp,
    ),
    keyDownHandler: (_) => _selectTab(context, 1),
  ),
];
```
So to avoid this kind of loop in the future:
```
for (final hotKey in hotKeys) {
  hotKeyManager.register(
    hotKey.hotKey, 
    keyDownHandler: hotKey.keyDownHandler,
  );
}
```
I find this to be much cleaner:
```
for (final hotKey in hotKeys) {
  hotKey.register();
}
```